scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -941,24 +941,23 @@ private static void processOngoingHandshakeMessage(final HandshakeMessage messag
 	 * 
 	 * @param helloRequest
 	 * @param connection
-	 * @throws HandshakeException if the message to initiate the handshake with the peer cannot be created
+	 * @throws HandshakeException if the message to initiate the handshake with
+	 *             the peer cannot be created
 	 */
-	private void processHelloRequest(final HelloRequest helloRequest, final Connection connection) throws HandshakeException {
+	private void processHelloRequest(final HelloRequest helloRequest, final Connection connection)
+			throws HandshakeException {
 		if (connection.hasOngoingHandshake()) {
-			// TLS 1.2, Section 7.4 advises to ignore HELLO_REQUEST messages arriving while
-			// in an ongoing handshake (http://tools.ietf.org/html/rfc5246#section-7.4)
-			LOGGER.debug(
-					"Ignoring {} received from [{}] while already in an ongoing handshake with peer",
-					new Object[]{helloRequest.getMessageType(), helloRequest.getPeer()});
+			// TLS 1.2, Section 7.4 advises to ignore HELLO_REQUEST messages
+			// arriving while in an ongoing handshake
+			// (http://tools.ietf.org/html/rfc5246#section-7.4)
+			LOGGER.debug("Ignoring {} received from [{}] while already in an ongoing handshake with peer",
+					helloRequest.getMessageType(), helloRequest.getPeer());
 		} else {
+			// We do not support re-negociation as recommended in :
```

Comment:
typo? negotiation?

Reply:
Thx, It sounds more appropriate to use the English spelling than the French one :-D